react-debugging.html

```diff
@@ -79,10 +80,10 @@ <h1>Draw a Card</h1>
           }
           return(
             <>
-              <button onClick={}>
+              <button onClick={this.drawCard}>
```

Comment:
this.props.drawCard - calling on the name of the props object because this is a child component.